druntime/src/importc.h

```diff
@@ -106,6 +106,12 @@
 #define __builtin___strncat_chk(dest, src, len, x) strncat(dest,src,len)
 #define __builtin___strncpy_chk(dest, src, len, x) strncpy(dest,src,len)
 #define __builtin___sprintf_chk(s, flag, os, fmt, ...) sprintf(s, fmt, __VA_ARGS__)
+#define __builtin___snprintf_chk(s, c, flag, os, fmt, ...) snprintf(s, c, fmt, __VA_ARGS__)
+#define __builtin___sprintf_chk(s, flag, os, fmt, ...) sprintf(s, fmt, __VA_ARGS__)
```

Comment:
isn't line 110 a duplicate redefinition of line 108?

Reply:
fixed